2018-05-16T04:55:34.004Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '001' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:55:34.005Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '001' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '001' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-16T04:55:34.522Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'the@thegioididong.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:55:34.522Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'the@thegioididong.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'the@thegioididong.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:55:34.540Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'giao@giaohangnhanh.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:55:34.540Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'giao@giaohangnhanh.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'giao@giaohangnhanh.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:55:34.558Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'easymoney@easymoneyinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:55:34.558Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'easymoney@easymoneyinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'easymoney@easymoneyinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:55:34.631Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'sieu@sieuthidienmay.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:55:34.631Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'sieu@sieuthidienmay.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'sieu@sieuthidienmay.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:55:34.642Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'app@theapple.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:55:34.642Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'app@theapple.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'app@theapple.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:55:34.647Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'linh@vinhsang.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:55:34.647Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'linh@vinhsang.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'linh@vinhsang.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:55:34.732Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'sam@samsung.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:55:34.733Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'sam@samsung.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'sam@samsung.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:55:34.749Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'nga@glorious.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:55:34.749Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'nga@glorious.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'nga@glorious.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:55:34.856Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'chuyen@chuyenphatnhanh.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:55:34.856Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'chuyen@chuyenphatnhanh.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'chuyen@chuyenphatnhanh.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:57:03.692Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: The current identity has not been registered: linh@vinhsang.com)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:57:03.692Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity has not been registered: linh@vinhsang.com)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity has not been registered: linh@vinhsang.com)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-16T04:57:03.692Z ERROR   HLFConnection           :ping()                   {"message":"Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity has not been registered: linh@vinhsang.com)","stack":"Error: Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity has not been registered: linh@vinhsang.com)\n    at _checkRuntimeVersions.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:713:34)\n    at <anonymous>"}$
